src/main/java/com/image/Images.java

```diff
@@ -0,0 +1,24 @@
+package com.image;
+
+public interface Images extends Iterable<Image>{
+
+    void addImage(Image image);
+
+    default Extension getExtention(String url) {
```

Comment:
@PavelNejinskiy this should be a part of `Image` interface. The method should not expect anything to be passed as of now. The `url` instance can be taken from `this` if it is `Image`. Please move this method to `Image` interface